src/pf-tooltip/index.html

```diff
@@ -13,25 +19,39 @@
     </div>
     <hr>
 
+    <pf-tooltip id="tooltip-left-nv" placement="left" target-selector="#btn-left-nv">
+        Tooltip text on Left but not in view
+    </pf-tooltip>
+    <button id="btn-left-nv" type="button" class="btn btn-primary" aria-describedby="tooltip-left-nv">Left</button>
+
+    <div style="margin-left: 200px;">
     <pf-tooltip id="tooltip-left" placement="left" target-selector="#btn-left">
         Tooltip text on Left
     </pf-tooltip>
     <button id="btn-left" type="button" class="btn btn-primary" aria-describedby="tooltip-left">Left</button>
+    </div>
 
+    <hr>
+    <div id="div">
```

Comment:
I think the non-unique IDs are in the dropdown too, can you make them unique, or remove if they're not necessary